js/pages/header/TasksHandler.js

```diff
@@ -62,22 +62,27 @@
 	 */
 	$.pkp.pages.header.TasksHandler.prototype.appendToggleIndicator_ =
 			function(callingElement, event) {
-
 		var $header = this.getHtmlElement(),
-				$popover = $header.find('#notificationsPopover'),
-				$toggle = $header.find('#notificationsToggle');
+			$popover = $header.find('#notificationsPopover'),
```

Comment:
The JS linter will complain about this -- it expects a double-tab indent when continuing a previous line. Try `bash lib/pkp/tools/buildjs.sh -n` to run the linter locally.

Reply:
(Or you can create a PR that only updates the submodule with the specially-formulated commit name referring to your branch, i.e.: `pkp/pkp-lib#4984 Submodule update ##jonasraoni/i4984-fix-tasks-popover##` -- that'll cause Travis to run tests including a JS linting round. But that's a pretty painful way to work through JS linting errors.)

Reply:
@asmecher, I've updated the PR.
Just to confirm, it's expected that the `buildjs.sh` is emitting `1 error(s), 247 warning(s)`, right?

Reply:
I replaced the compiler.jar by the version 20120917 and looks like it fixed the problem =]